### PR TITLE
Fix Return parse error if not needed fields are found #1256

### PR DESCRIPTION
**Pull request by Anjali-NEC**
Fix issue #1256

### CODE REVIEW

File: src/lib/jsonParseV2/parseContextAttribute.cpp [hunk 1]
```diff
@@ -153,8 +153,8 @@ static std::string parseContextAttributeObject
     }
     else  // ERROR
     {
-      LM_W(("Bad Input (unrecognized property for ContextAttribute - '%s')", name.c_str()));
-      return "unrecognized property for context attribute";
+      alarmMgr.badInput(clientIp, "JSON parse error");
+      return "JSON parse error in context attribute";
```
Comment (kzangeli) [inline src/lib/jsonParseV2/parseContextAttribute.cpp:157]: This is not a "Parse Error" !
The current error message is quite good, and should not be changed.
Add a call to the alarm manager?  Sure, why not?
But, let's not lie about what went wrong!
This is NOT a parse error, but an unexpected (and not welcome) item

Comment (Anjali-NEC) [reply src/lib/jsonParseV2/parseContextAttribute.cpp]: Yes, you are right! I have reverted my changes in https://github.com/telefonicaid/fiware-orion/pull/4049/commits/5acdc12ec5d93282bd71423d4572167778759baa and added alarm for badinput.

**Comment by fgalan** [comment]
See comment at https://github.com/telefonicaid/fiware-orion/issues/1256#issuecomment-1025488343

**Comment by fgalan** [comment]
The parent issue of this PR (https://github.com/telefonicaid/fiware-orion/issues/1256) has been closed, so the PR will be also closed.

I understand this is not a problem as you have only covered one of the multiple parsing cases which need to be implemented :)

Thanks for your contributions!

**Pull request closed without merging**